packages/core/src/StateNode.ts

```diff
@@ -405,60 +407,18 @@ export class StateNode<
     return selectedTransition ? [selectedTransition] : undefined;
   }
 
-  public get initialStateValue(): StateValue | undefined {
```

Comment:
I believe this can be just retrieved by accessing `.initialState.value`, so I don't think it's worth keeping it around